src/pages/PullRequestPage/PullCoverage/routes/FilesChangedTab/FilesChanged/FilesChangedTable/FilesChangedTable.tsx

```diff
@@ -0,0 +1,365 @@
+
+  const table = useReactTable({
+    columns: getColumns({ pullId }),
+    data,
+    state: {
+      expanded,
+      sorting,
+    },
+    onSortingChange: setSorting,
+    onExpandedChange: setExpanded,
+    getCoreRowModel: getCoreRowModel(),
+    getSortedRowModel: getSortedRowModel(),
+    getExpandedRowModel: getExpandedRowModel(),
+    getRowCanExpand: () => true,
+  })
+
+  if (mostRecentCompare?.state === 'pending') {
+    return <Loader />
+  }
+
+  if (isEmpty(data) && !isLoading) {
+    return <p className="m-4">No files covered by tests were changed</p>
+  }
+
+  return (
+    <div className="filelistui" data-highlight-row="onHover">
+      <div>
+        {table.getHeaderGroups().map((headerGroup) => (
+          <div key={headerGroup.id} className="filelistui-thead">
+            {headerGroup.headers.map((header) => {
+              const isSorted = header.column.getIsSorted()
+
+              return (
+                <div
+                  key={header.id}
+                  data-sortable="true"
+                  {...{
+                    onClick: header.column.getToggleSortingHandler(),
+                  }}
+                  className={cs({
+                    'w-8/12': header.id === 'name',
+                    'w-1/12 flex justify-end': header.id !== 'name',
+                  })}
+                  {...(isNumericValue(header.id)
+                    ? {
+                        'data-type': 'numeric',
+                      }
+                    : {})}
+                >
+                  <div
+                    className={cs('flex gap-1 items-center', {
+                      'flex-row-reverse justify-end': header.id === 'name',
+                    })}
+                  >
+                    <span
+                      className="text-ds-blue-darker"
+                      data-sort-direction={isSorted}
+                    >
+                      <Icon name="arrowUp" size="sm" />
+                    </span>
+                    {flexRender(
+                      header.column.columnDef.header,
+                      header.getContext()
+                    )}
+                  </div>
+                </div>
+              )
+            })}
+          </div>
+        ))}
+        {isLoading ? (
+          <Loader />
+        ) : (
+          table.getRowModel().rows.map((row, i) => (
+            <Fragment key={i}>
+              <div className="filelistui-row">
+                {row.getVisibleCells().map((cell) => {
+                  return (
+                    <div
+                      key={cell.id}
+                      {...(isNumericValue(cell.column.id)
+                        ? {
+                            'data-type': 'numeric',
+                          }
+                        : {})}
+                      className={cs({
+                        'w-8/12': cell.column.id === 'name',
+                        'w-1/12 justify-end	flex': cell.column.id !== 'name',
```

Comment:
Just being super picky here, but what's going on with that massive space?

Reply:
<img width="1257" alt="Screenshot 2024-01-26 at 11 46 36 AM" src="https://github.com/codecov/gazebo/assets/91732700/5a0d3ca9-12ea-4b57-8567-5f033d3a3071">

it's not really there

Reply:
suuuuuuure, keep telling yourself that 😝

Reply:
It actually looks like it's rendering as a tab character in the GH UI.